third_party/bazel/cuda_supplement/cuda_supplement_configure.bzl

```diff
@@ -8,23 +8,19 @@ def _create_dummy_repository(repo_ctx):
         "%{IF_HAS_CUBLASLT}": "False",
```

Comment:
Nit: I think we can use a more general env name, such as `CUDA_HOME`. I mean `TF_CUDA_HOME` -> `CUDA_HOME`